input/pagecontent/future.md

```diff
@@ -29,4 +29,9 @@ The approach to develop and mature AU Core is outlined below making reference to
     <img src="maturing.png" alt="Maturing the AU Core baseline" style="width:80%"/>
   </div>
 *Figure 3: Maturing the AU Core baseline*
-<br/>
\ No newline at end of file
+<br/>
+
+### AU Core Profiles Roadmap 
+
+#### AU Core MedicationStatement 
+The development of the AU Core MedicationStatement profile is planned for the AU Core R2 scope, following the TDG's strategic decision to prioritise the AU Core MedicationRequest profile for R1. Should time allow, the TDG plans to work on the AU Core MedicationStatement within R1 as well.
```

Comment:
Suggest instead
Development and testing of AU Core MedicationStatement profile to support AUCDI Medication Use Statement is planned for the AU Core R2 scope.

Reply:
Done

Reply:
still says the words scope after R2. But we can fix this later.